talk/owt/sdk/ic/icmanager.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) <2021> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "talk/owt/sdk/ic/icmanager.h"
+
+#include "inference_engine.hpp"
+#include "talk/owt/sdk/ic/backgroundblur.h"
+
+namespace owt {
+namespace ic {
+ICManager::ICManager() : core(new InferenceEngine::Core()) {
+}
+
+bool ICManager::InitializeInferenceEngineCore(
+    const std::string& plugins_xml_path) {
+  try {
+    core->RegisterPlugins(plugins_xml_path);
+    return true;
+  } catch (std::exception &) {
```

Comment:
You don't have issue building this? we by default build with -fno-exception. if you want to use this, cflags_cc should remove -fno-exception

Reply:
OpenVINO C++ API use exception semantics, may I enable exception handling just for ic plugin?

Reply:
yup.